Internship2022TeamPatri/Internship2022TeamPatri/Views/Auth/Register/Register.storyboard

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you have both storyboard and xib?

Reply:
In xib I have a custom view for the fields with label and text field. In storyboard I have the 5 custom views, the title, backgroun and the button

Reply:
I think you should move the storyboard into the same folder with RegisterViewController file

Reply:
I don't think we need a storyboard for this screen.